ssh-key/src/error.rs

```diff
@@ -45,7 +46,7 @@ pub enum Error {
 
     /// ECDSA key encoding errors.
     #[cfg(feature = "ecdsa")]
-    Ecdsa(sec1::Error),
+    Ecdsa(Arc<sec1::Error>),
```

Comment:
cc @tarcieri 
This comes from https://github.com/RustCrypto/formats/pull/1328

Reply:
We can revert that if it's causing too much of a problem

Reply:
This isn't the first place where that caused me problems. I think I might want to do that.